docker-compose.yml

```diff
@@ -54,6 +50,21 @@ services:
       interval: 5s
       timeout: 5s
       retries: 50
+  server:
+    image: nginx:stable-alpine
+    container_name: ft_transcendence-nginx
+    depends_on:
+      - web
+      - api
+    restart: always
+    networks:
+      - app
+    volumes:
+      - ./templates:/etc/nginx/templates
+    ports:
+      - 8080:80
+    environment:
+      NGINX_ENTRYPOINT_QUIET_LOGS: 1
```

Comment:
I do not necessarily agree with removing the logs

Reply:
That is only entrypoint logs, not nginx logs. Was trying to prevent the overflood of logs from nginx (every single request is logged). Doesn't matter if you want it back